reference/toaztr.v1.yaml

```diff
@@ -199,6 +206,13 @@ components:
       schema:
         type: string
       description: Curseur à partir duquel la prochaine page débute
+    simulationId:
```

Comment:
to be removed

Reply:
It's linked to the other comment: needed only if the get on '/v1/billing/cases/{caseId}/plans-parameters/{planId}/simulations/{simulationId}' is needed